docs/docs-beta/docs/getting-started/installation.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Installing Dagster"
+description: "Learn how to install Dagster"
+sidebar_position: 20
+sidebar_label: "Installation"
+---
+
+# Installing Dagster
+
+To follow the steps in this guide, you'll need:
+
+- To install Python 3.8 or higher. **Python 3.11 is recommended**.
+- To install pip, a Python package installer
```

Comment:
We could call out `uv` here too, since it's that's what the cool kids are doing.
```suggestion
- To install pip or uv for installing Python packages
```

Reply:
Edit: I might take this back, since it will impact the rest of the tutorial, for example:

```
uv venv
```

```
uv pip install
```